frontend/src/component/Chatbot.tsx

```
@@ -1,185 +1,220 @@
+                            data.message.message}
+                          {data.message.messageType == "NEWS" &&
+                            data.message.news &&
+                            data.message?.news?.data && (
+                              <CardNews newsList={data.message?.news?.data} />
+                              //   <></>
+                            )}
+                        </p>
+                      </div>
+                    </>
+                  )}
+                </>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)